src/pages/EventsPage.tsx

```
@@ -23,433 +26,300 @@ import {
-                </motion.div>
-              );
-            })}
+                </div>
+
+                <div
+                  className="prose prose-invert max-w-none mb-10 text-white/70"
+                  dangerouslySetInnerHTML={{ __html: content }}
+                />
+
+                <div className="mt-auto flex flex-col sm:flex-row gap-4">
+                  <a href="#" className="btn btn-primary flex-1 flex items-center justify-center gap-2 py-4 text-lg">
```

Biome (2.3.14)

[error] Provide a valid value for the attribute href.

The href attribute should be a valid a URL
Check this thorough explanation to better understand the context.

(lint/a11y/useValidAnchor)